src/jsonpatcherproxy.js

```diff
@@ -134,7 +134,7 @@ const JSONPatcherProxy = (function() {
     } else {
       if (Array.isArray(target) && !Number.isInteger(+key.toString())) {
         /* array props (as opposed to indices) don't emit any patches, to avoid needless `length` patches */
-        if(key !== 'length') {
+        if(key.toString() !== 'length') {
```

Comment:
Why you have to convert it?

Reply:
Because key is not always a string. It can be a symbol. See two lines above. But I admit that "length" is always a string.

Reply:
I removed the `toString` and laxed the comparision just in case.

Reply:
I thought if it's a symbol, it would still return `true` for `key !== 'length'`, so there is no reason to relax the comparison. Am I wrong?

Reply:
I don't think it would. Because, for example, `typeof Symbol.iterator` is `symbol` while type of `'length'` is `string`, Strict equality would return false.

Reply:
But we are talking about strict inequality. So that's actually what we want.

Reply:
You're right. But we start from different premises. I laxed the `!=` because I thought it can happen then a Symbol can occur that points to array length. As in `arr[Symbol.length]` in which case we don't want to warn. 

But then again I don't think this will ever happen.